objectserver/auditor.go

```diff
@@ -390,7 +394,10 @@ func (a *Auditor) auditSuffix(suffixDir string) {
 		a.totalBytes += bytesProcessed
 		rateLimitSleep(a.passStart, a.totalPasses, a.filesPerSecond)
 		rateLimitSleep(a.passStart, a.totalBytes, a.bytesPerSecond)
-		if err != nil {
+		if err == expiredObject {
+			a.logger.Debug("Removing expired object", zap.String("hashDir", hashDir))
```

Comment:
if the indexdb has the ExpireObjects thing why is the auditor doing it too? it just removes the files and doesn’t tell indexdb? that seems weird

Reply:
This part of the auditor is just for the swift backend.

Reply:
oooohhhh...